bricks/template/__brick__/{{project_name.snakeCase()}}/pubspec.yaml

```diff
@@ -33,8 +33,8 @@ dependencies:
   intl: ^0.17.0
   json_annotation: ^4.5.0
   package_info_plus: ^1.4.2
-  permission_handler: ^8.3.0
   retrofit: ^3.0.1+1
+{{#add_permission_handler}}{{{ _pubspec_dependencyyaml }}}{{/add_permission_handler}}
```

Comment:
Would it be possible to move the placeholder to line 36? It'd be better if we keep the dependencies sorted by alphabet.

Reply:
Agree, since the template optional features could be generated, we should not keep optional dependencies/components at the end for grouping anymore

Reply:
@hoangnguyen92dn thanks, updated in [de4d201](https://github.com/nimblehq/flutter-templates/pull/203/commits/de4d201fb66a46cf82bfb847c66ddeeefc349f8b)